src/plugin/renderer/texture_manager.cpp

```diff
@@ -20,8 +20,12 @@ namespace wmr
 		// Create a texture pool using the D3D12 Wisp renderer
 		m_texture_pool = dynamic_cast<const ViewportRendererOverride*>(MHWRender::MRenderer::theRenderer()->findRenderOverride(settings::VIEWPORT_OVERRIDE_NAME))->GetRenderer().GetD3D12Renderer().CreateTexturePool();
 
+		LOG("Attempting to get a reference to the renderer.");
```

Comment:
It actually already tried to get a reference when this LOG is called 😉 Doesn't matter anyway

Reply:
Fixed it!